docs/source/tutorials/audio.ipynb

```diff
@@ -346,6 +346,7 @@
     "df = pd.DataFrame(file_paths, columns=[\"wav_audio_file_path\"])\n",
     "df[\"label\"] = df.wav_audio_file_path.map(lambda x: int(Path(x).parts[-1].split(\"_\")[0]))\n",
     "# Note: Classes must be represented as integer indices 0, 1, ..., num_classes - 1\n",
+    "# Eg. for dataset with 7 examples from 3 classes, labels might be: np.array([2,0,0,1,2,0,1])\n",
```

Comment:
labels just needs to be an array_like, so we could simplify this a tiny bit, omit the `np.array` part, and write just `[2,0,0,1,2,0,1]`

Reply:
isn't numpy array preferred for efficiency?  Example should show most preferable input format right?

Reply:
I doubt it affects efficiency much. Showing the np.array version seems fine.

Reply:
I meant efficiency of users' overall code beyond cleanlab because they probably already have labels in pandas/numpy/tensor format